### ISSUE TITLE
In Excel, NVDA doesn't report status formating when toggling it

### ISSUE THREAD

**Issue by Hatoon-fel** [nvaccess/nvda#15684]
[excel with nvda.txt](https://github.com/nvaccess/nvda/files/13178121/excel.with.nvda.txt)
### Steps to reproduce:
open Excel new worksheet.
write anything in any cell then select it to change its format.
Press Ctrl+b or Ctrl+2 or Ctrl+4 or Ctrl+I to togle any formatting atribute.
Press NVDA+F to get report of format to be sure that the formatting aplied.
### Actual behavior:
NVDA silent, and will not report anything!!
### Expected behavior:
NVDA should announce the status like: "Bolt on", "Italic off" Etc as described in isssue #14923
#### NVDA installed/portable/running from source:
installed and Portable
#### NVDA version:
NVDA2023.3RC2
#### Name and version of other software in use when reproducing the issue:
Microsoft 365 MSO (Version 2310 version update 16.0.16924.20054) 32 Bit
#### Does the issue still occur after restarting your computer?
Yes

**Comment by Hatoon-fel** [comment]
Hello.
I'm using The Arabic interface for Windows, Ofice, and NVDA. But, i tried this with NVDA English interface, and nothing changed!
I tried with UIA enabled and disabled, the result doesn't change!
Thank you.
Hatoon

**Comment by CyrilleB79** [comment]
Could you provide a log in DEBUG mode, in case we see something unexpected.

Also, do you confirm me that you are not currently editing the cell when you press the formatting shortcuts? I.e. you have pressed Enter after having written anything in the cell?

**Comment by Hatoon-fel** [comment]
It doesn't work when i'm not in edit mode as well!! instead NVDA reads cell content when i press the shortcut  E.G i wrote: 'hello', then pressing Ctrl+b nvda will say: Hello!!!

**Comment by CyrilleB79** [comment]
OK. I would need a debug log then.

For this:
* Press NVDA+Q
* In the combo-box, select "Restart with debug logging enabled"
* Press Enter
* When NVDA has restarted, reproduce the issue in Excel
* Then, press NVDA+F1 to open the log
* Press control+S and save your log

Thanks

**Comment by Hatoon-fel** [comment]
I already did that. Please see the log in my reply above.

**Comment by CyrilleB79** [comment]
> I already did that. Please see the log in my reply above.

Oh sorry. I have found it now.

Unfortunately, you have taken the log with Excel using UIA in Advanced settings. Could you please take the same log disabling this option in Advanced settings? And also if possible with add-ons disabled.

To start NVDA without add-ons and with log enabled:
* Type Windows+R
* Type "nvda --disable-addons --debug-logging" (without quotes) and press enter

Thanks.

**Comment by Hatoon-fel** [comment]
Hello again!
it's working now with UIA disabled!
So, this feature dosn't work when UIA enabled or in editing mode, Is that right??
Thank you.

**Comment by CyrilleB79** [comment]
> Hello again! it's working now with UIA disabled! So, this feature dosn't work when UIA enabled or in editing mode, Is that right?? Thank you.

Yes, that's right.

I guess that you usully work with UIA disabled in Excel, don't you? If yes, I'd suggest that you update the initial description of this issue focusing around the issue while editing.

For completeness, you may also open another issue with the information regarding full cell formatting with UIA so that it is tracked.

It's easier to have two separate issues to track these two issues because they will be fixed separately. Thanks.

**Comment by seanbudd** [comment]
Closing as a duplicate of #14954

**Comment by CyrilleB79** [comment]
@seanbudd, this issue is not a duplicate of #14954.

#14954 is about reporting formatting when navigating between cells with arrow keys or pressing NVDA+F.

The current issue is about reporting formatting when pressing Excel's native formatting commands (control+B, control+U, etc.)

Thus I am reopening.

@Hatoon-fel, as asked in https://github.com/nvaccess/nvda/issues/15684#issuecomment-1784145036, editing the initial description focusing on the cell's edition use case is still expected, as well as opening another issue focusing on UIA. Thanks.

**Comment by CyrilleB79** [comment]
@Hatoon-fel or @seanbudd, could you edit the title of this issue as follows:
"While editing an Excel cell, NVDA doesn't report status formating when toggling it"

**Comment by CyrilleB79** [comment]
@seanbudd or @Hatoon-fel could you please update the title and the initial description to specify explicitly that this issue deals with toggling formatting shortcuts used while the cell is in edition mode?
